src/api/parser/package.json

```diff
@@ -25,14 +28,18 @@
     "linkifyjs": "3.0.5",
     "normalize-url": "6.1.0",
     "rss-parser": "3.12.0",
-    "sanitize-html": "2.5.3"
+    "sanitize-html": "2.5.3",
+    "set-interval-async": "^2.0.3"
   },
   "engines": {
     "node": ">=14.0.0"
   },
   "devDependencies": {
+    "@senecacdot/eslint-config-telescope": "1.1.0",
     "babel-jest": "27.5.1",
     "env-cmd": "10.1.0",
-    "nodemon": "2.0.15"
```

Comment:
Why would you remove `nodemon` in here?

Reply:
I didn't remove it, I switched them in a-z order.